topydo/lib/Utils.py

```diff
@@ -94,3 +94,20 @@ def inner():
             get_terminal_size.getter = inner
 
         return get_terminal_size.getter()
+
+
+def first_to_upper(p_str):
+    """ Returns p_str with first letter capitalized. """
+    first_capped = p_str[:1].upper() + p_str[1:]
+    return first_capped
+
+
+def translate_key_to_cfg(p_key):
+    """
+    Translates urwid key event to form understandable by topydo config parser.
+    """
+    key = first_to_upper(p_key)
```

Comment:
Oh, totally forgot about that function. Time to re-read docs for built-in types :) Fixed with recent rebase.